pkg/filterengine/filters/object_annotation_checker.go

```diff
@@ -82,6 +107,101 @@ func isObjectNotifDisabled(obj metaV1.ObjectMeta) bool {
 	return false
 }
 
+// isObjectNamespaceNotifEnabled checks annotation botkube.io/notification on the namespace
+// annotation botkube.io/notification enable/disable all object's event notification in the namespace
+func isObjectNamespaceNotifEnabled(obj metaV1.ObjectMeta) bool {
+
+	namespace := obj.Namespace
+	nsObjectNotifEnabled, found := NamespaceCache.Get(namespace)
+
+	if !found {
+		log.Warnf("Cache Miss!! Namespace '%s' not found in the cache!", namespace)
+
```

Comment:
In go, we try to avoid else block for better readability. 
You can return if found, and let move rest of the code outside else
```
	if found {
	        log.Infof("Cache Hit!! Namespace '%s' found in the cache!", namespace)
	        return nsObjectNotifEnabled
	}
       // rest of the logic if not found
```